style.css

```diff
@@ -1,4 +1,4 @@
-@import url("https://fonts.googleapis.com/css2?family=Inter&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
```

Comment:
- **[OPTIONAL]** Currently the scrolling when clicking a nav link to take to the corresponding section is happening too fast and enhances the scrolling let's say by implementing a smooth animation you can easily add the below code to the CSS file, and enjoy the experience 👍🏼 

```css
html {
  scroll-behavior: smooth;
}
```